simulation/simple_sensor_simulator/src/vehicle_simulation/ego_entity_simulation.cpp

```diff
@@ -97,22 +97,28 @@ auto EgoEntitySimulation::makeSimulationModel(
   const traffic_simulator_msgs::msg::VehicleParameters & parameters)
   -> const std::shared_ptr<SimModelInterface>
 {
+  rclcpp::Node node{"get_parameter", "simulation"};
+  auto get_parameter = [](rclcpp::Node & node, const std::string & name, auto value = {}) {
+    node.declare_parameter<decltype(value)>(name, value);
+    node.get_parameter<decltype(value)>(name, value);
+    return value;
+  };
```

Comment:
```suggestion
  auto node = rclcpp::Node("get_parameter", "simulation");

  auto get_parameter = [&](const std::string & name, auto value = {}) {
    node.declare_parameter<decltype(value)>(name, value);
    node.get_parameter<decltype(value)>(name, value);
    return value;
  };
```

Reply:
Indeed it looks better as there's no need to pass the “node” every each time. :bow: